cocotb/_build_libs.py

```diff
@@ -74,6 +95,7 @@ def finalize_options(self):
         super().finalize_options()
 
         for ext in self.extensions:
+            ext.library_dirs.append(os.path.join(self.build_lib, "cocotb", "libs"))
             ext.library_dirs.append(
                 os.path.join(self.build_lib, os.path.dirname(ext._full_name))
             )
```

Comment:
Is this one ever different to the one above?

Reply:
Yes. Not needed anymore.